### PR TITLE
feat(doc): Dedicated install doc

### PR DESCRIPTION
**Pull request by jsirianni**
### Proposed Change

Added a dedicated installation doc covering the following:
- server install (linux)
- client install (linux, mac, windows)
- collector install via ui
- collector install via cli
- collector install manually

##### Checklist
- [ ] Changes are tested
- [ ] CI has passed

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+1. Download a Windows Release: https://github.com/observIQ/bindplane-op/releases/download/v0.5.0/bindplane-v0.5.0-windows-amd64.zip
+2. Extract the zip file
+3. Run `bindplanectl.exe` from command prompt or PowerShell.
+4. Optionally [add the executable to your path](https://docs.microsoft.com/en-us/previous-versions/office/developer/sharepoint-2010/ee537574(v=office.14))
+
+Once installed, create a [client profile](./configuration.md#client-profiles).
+
+### macOS
+
```
Comment (andykellr) [inline docs/install.md:148]: Any reason not to recommend homebrew?

File: docs/install.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+### Web Interface
+
+1. Log into the BindPlane web interface
+2. Click "Install Agents"
+3. Choose your target operating system
+
+Copy the command generated by the web interface, and run it on the system you wish to install the collector on.
+
+After running the collector install command, the collector will appear in the web interface under the "agents" tag.
```
Comment (andykellr) [inline docs/install.md:177]:
```suggestion
After running the collector install command, the collector will appear in the web interface under the Agents tab.
```

File: docs/install.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+### Bindplane CLI
+
+If you wish to use `bindplanectl` to generate your install command (instead of using the web interface), you can run the following
```
Comment (andykellr) [inline docs/install.md:181]:
```suggestion
If you wish to use `bindplanectl` to generate your install command, you can run the following
```

File: docs/install.md (hunk 1 above)
Comment (jsirianni) [reply docs/install.md:148]: I keep forgetting that homebrew will install bindplanectl, I will update.

**Review by andykellr — APPROVED** [review]
We can resolved the Collector / Agent discussion in another PR.

**Pull request merged**